frontend/src/components/BarrierQuestionsView.tsx

```diff
@@ -3,20 +3,22 @@ import * as React from 'react'
 
 import { Typography, Divider } from '@equinor/eds-core-react'
 
-import { Barrier, Question } from "../../../api/models"
-import QuestionAndAnswerFormWithApi from '../../../components/QuestionAndAnswer/QuestionAndAnswerFormWithApi'
-import { barrierToString } from '../../../utils/EnumToString'
-import { getAzureUniqueId } from '../../../utils/Variables'
+import { Barrier, Question } from "../api/models"
+import { barrierToString } from '../utils/EnumToString'
+import QuestionAndAnswerFormWithApi from './QuestionAndAnswer/QuestionAndAnswerFormWithApi'
+import { getAzureUniqueId } from '../utils/Variables'
 
 interface BarrierQuestionsViewProps
 {
     barrier: Barrier
     questions: Question[]
+    showAnswerSummaryButton: boolean
+    onQuestionSummarySelected: (question: Question) => void
```

Comment:
Maybe remove the redundant prop?

```suggestion
    onQuestionSummarySelected?: (question: Question) => void
```

Reply:
Yeah, I was thinking about this too. But that means deciding if the button is visible based on whether or not that prop is defined. I like that, but I can also see it could be confusing, what do you think?

Reply:
Otherwise there will be a coupling between those two props. I think it seems better like you just suggested.

Reply:
:+1: